TestSuites/RDP/RDPSUTControlAgent/Python/RDPSUTControlAgent.py

```diff
@@ -0,0 +1,276 @@
+            self.testcase_name = b''.join(self.testcase_name)
+        start = header_length + name_length
+        self.request_id, help_message_length = unpack(
+            '<hi', bytes(request[start:start + 6]))
+        if help_message_length:
+            start += 6
+            self.help_message = unpack(
+                '<%sc' % help_message_length, request[start:start + help_message_length])
+            self.help_message = b''.join(self.help_message)
+        start += help_message_length
+        payload_length = unpack('<i', request[start:start + 4])[0]
+        if payload_length:
+            start += 4
+            raw_payload = unpack('<%sc' % payload_length,
+                                 request[start:start + payload_length])
+            if self.command_id == command_id['START_RDP_CONNECTION']:
+                self.payload = Payload()
+                self.payload.decode(b''.join(raw_payload))
+            elif self.command_id == command_id['DISPLAY_UPDATE_MONITORS']:
+                self.monitor_action = unpack(
+                    '<i', request[start + payload_length:])
+            elif self.command_id == command_id['DISPLAY_UPDATE_RESOLUTION']:
+                print("DISPLAY_UPDATE_RESOLUTION")
+                # TODO: parse payload structure
+            elif self.command_id == command_id['TOUCH_EVENT_MULTIPLE']:
+                print("TOUCH_EVENT_MULTIPLE")
+                # TODO: parse payload structure
+            elif self.command_id == command_id['TOUCH_EVENT_SINGLE']:
+                print("TOUCH_EVENT_SINGLE")
+                # TODO: parse payload structure
+            elif self.command_id == command_id['BASIC_INPUT']:
+                print("BASIC_INPUT")
+                # TODO: parse payload structure
+            else:
+                self.payload = ''.join(raw_payload)
+                print(self.payload)
+
+
+def handle_connection(client_socket):
+    processes = []
+    while True:
+        request = client_socket.recv(BUFFER_SIZE)
+        if len(request) == 0:
+            continue
+
+        msg = Message()
+        msg.decode(request)
+        print("testcase: %s" % msg.testcase_name)
+        print("command: %s" % msg.command_id)
+        if msg.type != message_type['SUT_CONTROL_REQUEST'] or msg.testcase_id != testcase_id['RDP_TESTSUITE']:
+            print("message is not a control request")
+            continue
+
+        response = Message()
+        response.type = message_type['SUT_CONTROL_RESPONSE']
+        response.testcase_id = testcase_id['RDP_TESTSUITE']
+        response.command_id = msg.command_id
+        response.testcase_name = msg.testcase_name
+        response.request_id = msg.request_id
+        if msg.command_id == command_id['START_RDP_CONNECTION']:
+            print("make connection")
+            response.rc = result_code['SUCCESS']
+            client_cmd = ["/usr/bin/xfreerdp"]
```

Comment:
In the java version, we use a config file to specify the commands.

IMHO, we can use the similar method here to avoid hard-coding the commands in .py file itself.

Reply:
Will do.